### PR TITLE
[17.0][MIG] account_banking_sepa_credit_transfer: Migration to 17.0 

### PR DESCRIPTION
**Pull request by trisdoan**
Depends on:

- [account_banking_pain_base] <https://github.com/OCA/bank-payment/pull/1231>

### CODE REVIEW

**Comment by sbidoul** [comment]
Tests and runboat are now green here, following the new release of `whool`.

**Comment by pedrobaeza** [comment]
You can rebase now and remove the PR commit.

File: account_banking_sepa_credit_transfer/README.rst [hunk 1]
```diff
@@ -120,11 +120,12 @@ Contributors
 -  `Trobz <https://trobz.com>`__:
 
    -  Dung Tran <dungtd@trobz.com>
+   -  Tris Doan <tridm@trobz.com>
 
 Other credits
 -------------
 
-The migration of this module from 15.0 to 16.0 was financially supported
+The migration of this module from 16.0 to 17.0 was financially supported
```
Comment (pedrobaeza) [inline account_banking_sepa_credit_transfer/README.rst:128]: Is this still true?

Comment (trisdoan) [reply account_banking_sepa_credit_transfer/README.rst:128]: Hello @pedrobaeza, yes it is

**Review by pedrobaeza — APPROVED** [review]

**Review by luc-demeyer — APPROVED** [review]

**Pull request merged**